org-status.py

```diff
@@ -1,74 +1,86 @@
 from os import environ
-from multiprocessing import Pool
+from multiprocessing.dummy import Pool
 from argparse import ArgumentParser
 
-import requests
 from termcolor import colored
-from IGitt.GitHub.GitHub import GitHub, GitHubToken
-from IGitt.GitHub.GitHubOrganization import GitHubOrganization
 
-error = 'error'
-unknown = 'unknown'
-failing = 'failing'
-passing = 'passing'
+from status_providers import Status
+from org_hosts import get_all_supported_hosts
 
-def process_travis_result(html):
-  html = html.lower()
 
-  for status in [passing, failing, unknown, error]:
-    if status in html:
-      return status    
+def get_host_token(host_name):
+    return environ['{}_TOKEN'.format(host_name.upper())]
 
-def get_travis_status(repo, branch='master'):
-  url = 'https://api.travis-ci.org/coala/{repo}.svg?branch={branch}'.format(
-          repo=repo, branch=branch)
 
-  svg_result = requests.get(url)
-  svg = svg_result.text
+def get_org_host(host_org):
+    parts = host_org.strip().split(':')
 
-  return process_travis_result(svg)
+    # defaults to gh
+    if len(parts) == 1:
+        parts = ['github', parts[0]]
```

Comment:
more useful if it doesnt default to `github` , but instead does multiple hosts.

so we can run it once for coala and see results both gitlab and github.

e.g. `org-status coala` , or also good `org-status :coala`

the colon is only needed to reliably parse orgs who are also hosters

- http://gitlab.com/github and https://gitlab.com/travis-ci are squatted names.
- http://github.com/gitlab and https://github.com/gitlab-ci are real
- https://github.com/travis-ci is real